diff --git a/modules/custom/utexas_hero/utexas_hero.module b/modules/custom/utexas_hero/utexas_hero.module
index 1a150b08..2ff8c13f 100644
--- a/modules/custom/utexas_hero/utexas_hero.module
+++ b/modules/custom/utexas_hero/utexas_hero.module
@@ -218,7 +218,7 @@ function _utexas_hero_add_new_hero_image_form_elements($form, $entity_type) {
     'utexas_hero_1' => '<span>Style 1:</span> <a href="https://demo.drupalkit.its.utexas.edu/hero-styles/1" target="_blank">Bold heading & subheading on burnt orange background<span class="ut-cta-link--external"></span></a>',
     'utexas_hero_2' => '<span>Style 2:</span> <a href="https://demo.drupalkit.its.utexas.edu/hero-styles/2" target="_blank">Bold heading on dark background, anchored at base of media<span class="ut-cta-link--external"></span></a>',
     'utexas_hero_3' => '<span>Style 3:</span> <a href="https://demo.drupalkit.its.utexas.edu/hero-styles/3" target="_blank">White bottom pane with heading, subheading and burnt orange call to action<span class="ut-cta-link--external"></span></a>',
-    'utexas_hero_4' => '<span>Style 4:</span> <a href="https://demo.drupalkit.its.utexas.edu/hero-styles/4" target="_blank">Centered image with dark bottom pane containing heading, subheading and call-to-action<span class="ut-cta-link--external"></span></a>',
+    'utexas_hero_4' => '<span>Style 4:</span> <a href="https://demo.drupalkit.its.utexas.edu/hero-styles/4" target="_blank">Large image floated right, gray color block on left, white text box skewed left of center with heading, subheading, burnt orange call to action<span class="ut-cta-link--external"></span></a>',
     'utexas_hero_5' => '<span>Style 5:</span> <a href="https://demo.drupalkit.its.utexas.edu/hero-styles/5" target="_blank">Medium image, floated right, with large heading, subheading and burnt orange call-to-action<span class="ut-cta-link--external"></span></a>',
   ];
   // Add any view modes not specified above.
